### PR TITLE
[android] - update changelog for v8.0.0-beta.1

### PR DESCRIPTION
**Pull request by tobrun**
PR to update the changelog for v8.0.0-beta.1

### CODE REVIEW

File: platform/android/CHANGELOG.md [hunk 1]
```diff
@@ -3,14 +3,34 @@
...
+## 8.0.0-beta.1 - May 16, 2019
+
+### Features
+ - Option to provide custom values for location camera transition [#14606](https://github.com/mapbox/mapbox-gl-native/pull/14606)
+ 
+### Bugs
+ - Expose trackingGesturesManagement attribute [#14659](https://github.com/mapbox/mapbox-gl-native/pull/14659)
+ - Harden onLowMemory call, use atomic boolean for thread safe management of the render surface [#14622](https://github.com/mapbox/mapbox-gl-native/pull/14622) 
+ - Normalize previous rotation values of the location animator [#14626](https://github.com/mapbox/mapbox-gl-native/pull/14626)
+ - Remove missing image log [#14600](https://github.com/mapbox/mapbox-gl-native/pull/14600)
```
Comment (LukasPaczos) [inline platform/android/CHANGELOG.md]: This actually was released with `mojito`.

Comment (osana) [inline platform/android/CHANGELOG.md]: This one was added for 7.4.0

Comment (tobrun) [reply platform/android/CHANGELOG.md]: weird that it's not showing in the commit itself https://github.com/mapbox/mapbox-gl-native/commit/ea50c776cd16a95a84718fcdcec608c407977068, normally it should be included in the tag.

Comment (LukasPaczos) [reply platform/android/CHANGELOG.md]: I think you're looking for this commit https://github.com/mapbox/mapbox-gl-native/commit/82206fb1fb6d6f06eea90d9717b2fad230ba5e12. Isn't tag only displayed on the actually tagged commit?

File: platform/android/CHANGELOG.md [hunk 2]
```diff
@@ -3,14 +3,33 @@
...
+ 
 ## 7.4.0 - May 15, 2019
 ### Features
-- Revert SKU token logic [#14594](https://github.com/mapbox/mapbox-gl-native/pull/14594)
+ - Revert SKU token logic [#14594](https://github.com/mapbox/mapbox-gl-native/pull/14594)
 
 ### Bugs
-- Keep strong references to the FileSource/OfflineManager operations callbacks [#14601](https://github.com/mapbox/mapbox-gl-native/pull/14601)
-- Failed sprite requests do not block tiles rendering [#14605](https://github.com/mapbox/mapbox-gl-native/pull/14605)
-- Remove missing image log [#14600](https://github.com/mapbox/mapbox-gl-native/pull/14600)
```
Comment (LukasPaczos) [inline platform/android/CHANGELOG.md]: This should be kept.

File: platform/android/CHANGELOG.md (hunk 1 above)
Comment (tobrun) [reply platform/android/CHANGELOG.md]: :man_facepalming:

**Pull request merged**